packages/fast-components-class-name-contracts-msft/src/progress/index.ts

```diff
@@ -35,17 +35,17 @@ export interface ProgressClassNameContract extends BaseProgressClassNameContract
     progress_dot__2?: string;
 
     /**
-     * The indeterminate progress indicator 3 modifier
+     * @Deprecated 3.4.0

```

Comment:
Can you create a task for changing the names of `progress__dot`, `progress__dot__1`, `progress__dot__2` to be more semantically correct and link the task? As you mentioned in your other comment, the names don't make sense anymore so it would be fine to track that as an issue.

Reply:
Good Idea.

New Issue here:
https://github.com/Microsoft/fast-dna/issues/1215